focus-android/.buildconfig.yml

```diff
@@ -0,0 +1,149 @@
+projects:
+  app:
+    upstream_dependencies:
```

Comment:
Yes! The good new is: we have a lint task that turns red if this file gets out of sync. Cherry on the top: the lint task generates the diff you can easily apply locally (see above)

Reply:
Gotcha. Maybe we should add comment to say not to manually change this then?

Reply:
Bad news this time: comments are going to be removed the next time this file gets generated. It's not supported by PyYaml. I think it's okay because the lint task will explain what is expected.